Pinta.Core/Classes/DocumentSelection.cs

```diff
@@ -365,6 +365,35 @@ public void Invert (Surface surface, Core.Size imageSize)
 		MarkDirty ();
 	}
 
+	/// <summary>
+	/// Expands or contracts the selection by a specified amount.
+	/// </summary>
+	/// <param name="surface">
+	/// Surface for the selection path.
+	/// </param>
+	/// <param name='delta'>
+	/// The amount to expand the selection by. A positive value will expand the selection, and a negative value will contract the selection.
+	/// </param>
+	public void Offset (Surface surface, double delta)
+	{
+		// Remove any self-intersections from the selection polygons.
+		List<List<IntPoint>> simplePolygons = new List<List<IntPoint>> ();
```

Comment:
This can be shortened to `List<List<IntPoint>> simplePolygons = new ();`, and the same for `offsetPolygons` and `clipperOffset` below